### DISCUSSION TITLE
What are the supported features?

### DISCUSSION THREAD

**Discussion by dnestico** [mrlt8/docker-wyze-bridge#296] [General]
So glad I found this addon, it's awesome!!!

I just have one question which is, it says "supported features: 2", just wondering what exactly those are cuz curious, thanks!

<img width="1134" alt="Screen Shot 2022-02-20 at 2 37 11 AM" src="https://user-images.githubusercontent.com/3034555/154833245-cecdca20-2615-4176-bc77-a9ae1ff0abc3.png">
.

**Comment by mrlt8** [comment]
Is this in Home Assistant? I believe that message is coming from [camera.ffmpeg](https://www.home-assistant.io/integrations/camera.ffmpeg/), so you'd have to check that add-on, but if I had to guess, I would imagine the the video stream and the snapshot are the 2 features.

**Reply by dnestico** [reply]
Ah I see was just wondering

**Reply by NotaProgrammer26** [reply]
I've got stock firmware on the cameras and have not put in the changes for FFMPEG into my configuration for the bridge configuration.      It shows Supported Features of "0".   
My configuration is supporting HSL, RTSP, and Snapshots.   So, not sure if the  "features" is related to those capabilities.
stream: rtsp://homeassistantrpi4:8554/backyard
image: http://homeassistantrpi4:8123/local/backyard.jpg
entity_picture: /api/camera_proxy/camera.wyze_cam_backyard?token=b7………..
icon: mdi:image
friendly_name: Wyze Cam Backyard
supported_features: 0
